From f666d037eb10ee16873b19ba411d437ad6327e2a Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 20 Aug 2018 16:33:42 +0300
Subject: [PATCH] Makefile: push gaudi/dali-release pointer

Finally, delete old release pointer, and push new release pointer.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index f3adefe..ec2e079 100644
--- a/Makefile
+++ b/Makefile
@@ -72,7 +72,8 @@
 		git tag -afm "Release Habanalabs $(HL_FULL_VER)" $(HL_FULL_VER)
 	@repo forall $(RELEASE_REPOS) -c \
 		git tag -f $(RELEASE_TAG_POINTER)
-	@repo forall $(RELEASE_REPOS) -c git push origin $(HL_FULL_VER)
+	@repo forall $(RELEASE_REPOS) -c git push origin :$(RELEASE_TAG_POINTER)
+	@repo forall $(RELEASE_REPOS) -c git push origin $(HL_FULL_VER) $(RELEASE_TAG_POINTER)
 	@echo "Released: $(HL_FULL_VER)"
 
 .PHONY: hl-dali hl-gaudi
